src/redux/reducers/upload-reducer.js

```diff
@@ -11,7 +11,12 @@ const initState = {
   dataMapLength: 0,
   history: [
     // object returned by uploadedFileGenerator()
-  ]
+  ],
+  retentionYears: 5,
```

Comment:
Might want to set this to 0 or 1.

Reply:
It doesn't matter to me, I just left it at what it was before. Let me know if you want me to change it.

Reply:
I'm inclined to think that 1 should be our default value and the value for the initial state.  Most people who use the service at first are probably only going to want the minimum length of storage, since they are probably going to be people who are just testing the service out.

Reply:
Works for me, ill make that change.